doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1613 @@
+     */
+    SAI_NEXT_HOP_ATTR_TUNNEL_ID,
+...
+```
+
+**BUM and Unicast Filtering**
+
+New Bridge port attributes are defined to support:
+  	- Non-Designated Forwarder BUM Filtering
+  	- Unicast and BUM Filtering on Single Active Ethernet Segment
+
+Note: Traffic dropped due to this should not be counted against SAI_PORT_STAT_IF_OUT_DISCARDS i.e Interface Tx drop counters.
+
+```
+typedef enum _sai_bridge_port_attr_t
+{
+...
+
+    /**
+     * @brief Indicates if the bridge port is set to drop the Tunnel Terminated broadcast, unknown unicast and multicast traffic
+     * When set to true, egress BUM traffic will be dropped
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * @validonly SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_PORT or SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_SUB_PORT or SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_TUNNEL
+     */
+    SAI_BRIDGE_PORT_ATTR_TUNNEL_TERM_BUM_TX_DROP,
+
+    /**
+     * @brief Indicates if the bridge port is set to drop the ingress traffic
+     * When set to true, all ingress traffic will be dropped
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * @validonly SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_PORT or SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_SUB_PORT or SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_TUNNEL
+     */
+    SAI_BRIDGE_PORT_ATTR_RX_DROP,
+
+    /**
+     * @brief Indicates if the bridge port is set to drop the egress traffic
+     * When set to true, all egress traffic will be dropped
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * @validonly SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_PORT or SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_SUB_PORT or SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_TUNNEL
+     */
+    SAI_BRIDGE_PORT_ATTR_TX_DROP,
+...
+}
+```
+**Backup Nexthop Group support**
+
+A new Bridge port attribute (for type PORT) is introduced to specify the protection nexthop group ID.
+When the Bridge port is down, all of the traffic to the Bridge port is redirected to the protection nexthop.
+
+```
+typedef enum _sai_bridge_port_attr_t
+{
+...
+    /**
+     * @brief Associated protection bridge port nexthop group id
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_NEXT_HOP_GROUP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_BRIDGE_PORT_ATTR_BRIDGE_PORT_PROTECTION_NEXT_HOP_GROUP_ID,
+...
+}
+```
+
+**ISOLATION_GROUP**
+
+  - There is no change to the Isolation group and group member definition.
+  - Bridgeport of type TUNNEL will also now have the isolation group attribute set.
+  - Multiple Tunnel bridgeports can have the isolation group attribute set.
+
+
+[SAI PR 2058] (https://github.com/opencomputeproject/SAI/pull/2058) is raised for the above changes.
```

Comment:
This SONiC HLD refers to an abandoned [SAI PR 2058](https://github.com/opencomputeproject/SAI/pull/2058)
The actual EVPN MH SAI additions are in [PR 2084](https://github.com/opencomputeproject/SAI/pull/2084) which introduces SAI_BRIDGE_PORT_ATTR_BRIDGE_PORT_SET_SWITCHOVER. 

The expectation is that NOS attaches a PROTECTION_NEXT_HOP_GROUP_ID on LAG ports where protection is enabled. On the failure of these LAG ports, NOS triggers the failover, i.e., sets SAI_BRIDGE_PORT_ATTR_BRIDGE_PORT_SET_SWITCHOVER. The SAI implementation then ensures that the MAC addresses learnt on the failed LAG will now be forwarded on the PROTECTION_NEXT_HOP_GROUP associated with the failed LAG.

Please update the SONiC HLD to describe the sequence of SAI operations when a LAG fails.